doc/pages/options.asciidoc

```diff
@@ -273,7 +273,11 @@ are exclusively available to built-in options.
 
 *extra_word_chars* `codepoint-list`::
     a list of all additional codepoints that should be considered
-    as word character. This must be set on the buffer, not the window,
+    part of a word, for the purposes of the `w`, `b`, and `e` commands
+    (See <<keys#movement,`:doc keys movement`>>).
+    If this option is empty, Kakoune pretends it contains an
+    underscore, otherwise the value is used as-is.
```

Comment:
> Kakoune pretends it contains an underscore, otherwise the value is used as-is.

Maybe this is so `kak -f` works as expected (and I guess also `kak -n`, even though that seems less important).
If my guess is correct we might want ot add it:

```
If this option is empty, Kakoune pretends it contains an underscore. This
allows conventional word movement without loading configuration, like in `kak -f`.
```

(The "otherwise" part seems redundant.)